ecosystem/utils/submission_parser.py

```diff
@@ -16,6 +18,20 @@ def _clean_section(section: str) -> {str: str}:
     return (title, section)
 
 
+def _section_titles_to_ids(sections: dict[str, str]) -> dict[str, str]:
```

Comment:
I'm confused what `sections` is. It would be helpful to call it something like `section_titles_to_foo`.

Reply:
That's true, "sections" is very vague. In this case, each "section" is a markdown heading plus the content after it. I'll try and come up with better naming.